### DISCUSSION TITLE
Will Minecolonies use NeoForge or continue using Forge?

### DISCUSSION THREAD

**Discussion by shirahasan** [ldtteam/minecolonies#10066] [Ideas]
After the release of version 1.21, many mods have released NeoForge versions. I would like to ask if Minecolonies will release a NeoForge version, or will it continue to use Forge?

**Comment by Nightenom** [comment]
There are already 1.20.4 unsupported neo builds and 1.21is also neoforge. So the answer right now is neo only past 1.20.4

**Reply by shirahasan** [reply]
> There are already 1.20.4 unsupported neo builds and 1.21is also neoforge. So the answer right now is neo only past 1.20.4

Thank you for your response! I've been playing the 1.21 neoforge modpack recently, and it's fantastic that MineColonies can use neo. I'm really looking forward to the updates! Thanks again!
